documentation/grammars.md

```diff
@@ -0,0 +1,62 @@
+# Grammar Syntax Documentation
+
+Grammars in Pliant are defined by an bnf dialect similar to abnf and ebnf. Some additional control statements have been added for ignoring whitespace and specifying start productions.
+
+# Grammar of Grammars
+
+The supported language for grammars in Pliant is defined by the following BNF grammar. 
+
+```
+Definition          =   Block
+                        | Block Definition
+
+Block               =   Rule
+                        | Setting
+                        | LexerRule
+
+Rule                =   QualifiedIdentifier '=' Expression ';'
+
+Setting             =   SettingIdentifier '=' QualifiedIdentifier ';'
+
+LexerRule           =   QualifiedIdentifier '~' LexerRuleExpression ';'
+
+Expression          =   Term
```

Comment:
The language being defined is the Pliant grammar definition language which has a tilde.

I agree that the grammar used to describe that language should be Ebnf, Bnf or Abnf. I'll update the grammar by following the standards noted in #66 and #71